src/configAPIClient.ts

```diff
@@ -39,6 +40,25 @@ export interface IKitConfigs {
     excludeAnonymousUser: boolean;
 }
 
+export interface IFilteringAttributeValue {
```

Comment:
It looks like both interfaces are the exact same. Why do we need to differentiate? Can we just use `IFilteringAttributeValue` for both `filteringEventAttributeValue` and `filteringUserAttributeValue`?